charts/kserve-resources/templates/webhookconfiguration.yaml

```diff
@@ -4,7 +4,7 @@ kind: MutatingWebhookConfiguration
 metadata:
   name: inferenceservice.serving.kserve.io
   annotations:
-    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/serving-cert
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/kserve-webhook-server-cert
```

Comment:
the name is correct, see https://github.com/kserve/kserve/blob/master/charts/kserve-resources/templates/certificate.yaml#L4

Reply:
It is the certificate resource name not the underlying secret name

Reply:
for secret name cert manager support `cert-manager.io/inject-ca-from-secret`